apple/internal/partials/debug_symbols.bzl

```
@@ -303,13 +303,21 @@ def _bundle_dsym_files(
         found_binaries_by_arch.update(dsym_binaries)
 
     if found_binaries_by_arch:
-        output_files = _copy_dsyms_into_declared_bundle(
+        dsyms = _copy_dsyms_into_declared_bundle(
             actions = actions,
             debug_output_filename = dsym_output_filename,
             dsym_bundle_name = dsym_bundle_name,
             found_binaries_by_arch = found_binaries_by_arch,
             platform_prerequisites = platform_prerequisites,
         )
+        output_files.extend(dsyms)
+        dsyms_command = (" && ".join([
+            "cp \"{dsym_path}\" \"${{OUTPUT_DIR}}/Contents/Resources/DWARF/{dsym_bundle_name}\"".format(
+                dsym_path = dsym.path,
+                dsym_bundle_name = dsym_output_filename,
+            )
+            for dsym in dsyms
+        ]))
```

Is the name of `_copy_dsyms_into_declared_bundle` correct with your analysis/change? Should it be similar to `_generate_dsym_info_plist` with `_generate` as the prefix since that seems to be roughly the same operation at this point?

Nit suggestion:
```suggestion
        dsym_binaries = _copy_dsyms_into_declared_bundle(
            actions = actions,
            debug_output_filename = dsym_output_filename,
            dsym_bundle_name = dsym_bundle_name,
            found_binaries_by_arch = found_binaries_by_arch,
            platform_prerequisites = platform_prerequisites,
        )
        output_files.extend(dsym_binaries)
        dsyms_command = (" && ".join([
            "cp \"{dsym_path}\" \"${{OUTPUT_DIR}}/Contents/Resources/DWARF/{dsym_bundle_name}\"".format(
                dsym_path = dsym_binary.path,
                dsym_bundle_name = dsym_output_filename,
            )
            for dsym_binary in dsym_binaries
        ]))
```

---

Renaming makes sense, let me update